src/components/LiquidationPrice.js

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from "react";
+import Flex, {FlexItem} from "styled-flex-component";
+import ReactTooltip from "react-tooltip"
+import styled from "styled-components"
+import {device} from "../screenSizes"
```

Comment:
captialized d?

Reply:
it is an object 
same as before 
Capital letters are used for Class's in JS (and in some other languages as well)